### PR TITLE
corrections and modification for ta-99 keyboard

### PR DESCRIPTION
**Pull request by balajijagadesh**
[Key sequence for Tamil99.pdf](https://github.com/wikimedia/jquery.ime/files/411770/Key.sequence.for.Tamil99.pdf)
As per the specification of Tamil Nadu government specification of tamil 99 keyboard layout, 
I is for : 
" is for '
J is for ★
N is for ௐ
G is for ⚪
H is for ⚫

These changes are made in the key mapping. 

An issue regarding this was filed here https://github.com/wikimedia/jquery.ime/issues/442

A bug was also filed in phabricator regarding this here https://phabricator.wikimedia.org/T134233

For reference : http://www.unicode.org/L2/L2010/10318-tamil-nadu.pdf

Page 10 of 13 in appendix C

### CODE REVIEW

File: rules/ta/ta-99.js [hunk 1]
```diff
@@ -151,32 +151,32 @@
 			[ 'T', 'க்ஷ' ],
 			[ 'Y', 'ஸ்ரீ' ],
 			[ 'U', 'ற' ],
-			[ 'I', 'ன' ],
+			[ 'I', ':' ],
 			[ 'O', '[' ],
 			[ 'P', ']' ],
-			// '\\{', 'ஆ'],
+			// '\\{', 'ஆ'],
```
Comment (kartikm) [inline rules/ta/ta-99.js]: Trailing whitespace.

File: rules/ta/ta-99.js [hunk 2]
```diff
@@ -151,32 +151,32 @@
...
 			[ 'Z', '௳' ],
 			[ 'X', '௴' ],
 			[ 'C', '௵' ],
 			[ 'V', '௶' ],
 			[ 'B', '௷' ],
-			[ 'N', 'ல' ],
+			[ 'N', 'ௐ' ],
 			[ 'M', '/' ],
-
+
```
Comment (kartikm) [inline rules/ta/ta-99.js]: Extra spaces.

**Comment by kartikm** [comment]
Also, this pull request need changes in test cases.

**Comment by amire80** [comment]
Continued in #495.

**Pull request closed without merging**